www/widgets/DemoExampleMaker.js

```diff
@@ -291,6 +298,60 @@ class DemoExampleMaker extends Widget {
     this.$('[name="dem-url"]').style.display = 'block'
   }
 
+  _uploadJSON (dem) {
+    const fileInput = document.createElement('input')
+    fileInput.type = 'file'
+    fileInput.addEventListener('change', function () {
```

Comment:
hey, let's try to use our internal `nn.FileUploader` class to keep the code a little tighter (instead of rolling this out from scratch). u can find documentation for that [here](https://github.com/netizenorg/netnet-standard-library/blob/main/docs/API.md#FileUploader)